lib/index.js

```diff
@@ -24,9 +27,27 @@ mercutio.middleware = function({ resolveRoles = getRolesFromAuthorizationHeader,
 		req.identity.demand = (role) => {
 			if (!roles.isAny(role)) onDemandFail(req, res, next);
 		};
-
-		return next();
-	};
+		req.identity = { user: null, authenticated: false };
```

Comment:
Setting req.identity at this point will overwrite the setting above, which means `req.identity.roles` and `req.identity.is` are now gone. Please sanity check this. Provide unit tests.